components/AddCatCard.tsx

```diff
@@ -404,28 +446,32 @@ export const AddCatCard = ({
         <SelectDropdown
           data={['M', 'F']}
           defaultValue={localCat.sex}
-          buttonStyle={styles.genderButton}
-          buttonTextStyle={styles.genderButtonText}
-          dropdownStyle={styles.genderDropdown}
-          rowTextStyle={styles.genderRowText}
-          dropdownIconPosition="right"
-          renderDropdownIcon={(_selectedItem, _index) => (
-            <TextInput.Icon
-              icon="chevron-down"
-              color={theme.colors.text}
-              style={{ marginRight: 40 }}
-            />
+          renderButton={(selectedItem: any, isOpened: boolean) => (
+            <View style={styles.genderDropdownButtonContainer}>
```

Comment:
what is the type of `selectedItem` ?
From its usage it looks like it's a string